src/qcodes_contrib_drivers/drivers/Aviosys/IP_Power_9258S.py

```diff
@@ -92,3 +95,13 @@ def __init__(self, name: str, address: str, login_name: str, login_password: str
     # get functions
     def get_idn(self):
         return {'vendor': 'Aviosys', 'model': 'IP Power 9258S'}
+
+
+class Aviosys_IP_Power_9258S(AviosysIPPower9258S):
```

Comment:
I would suggest using the `@deprecated` decorator from typing_extensions to mark a class deprecated. You can see an example of how that works here https://github.com/microsoft/Qcodes/blob/main/src/qcodes/instrument_drivers/american_magnetics/AMI430.py

This will allow static type checkers to discover use of the deprecated class and mark import and use of it as striked out in vscode (and perhaps other editors)